modules/ting_search_carousel/ting_search_carousel.install

```diff
@@ -76,20 +37,36 @@ function ting_search_carousel_update_7003() {
 }
 
 /**
- * Ensure router path for carousel shortcut menu link is correct.
+ * Migrate front page carousel.
  */
-function ting_search_carousel_update_7004() {
```

Comment:
I don't think that we should remove old update functions. If not valid, leave them empty with a short description about why it's empty.